transports/transport-rest/transport-rest-vertx/src/main/java/io/servicecomb/transport/rest/vertx/accesslog/AccessLogHandler.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.transport.rest.vertx.accesslog;
+
+import io.vertx.core.Handler;
+import io.vertx.ext.web.RoutingContext;
+
+public interface AccessLogHandler extends Handler<RoutingContext> {
```

Comment:
what's the purpose of this interface?

Reply:
This interface has been removed. Now, it's implementation AccessLogHandlerImpl implements the interface Handler<RoutingContext> directly.

Reply:
The class AccessLogHandlerImpl is renamed to AccessLogHandler since this interface no longer exists.